src/org/labkey/snd/query/EventsTable.java

```diff
@@ -113,29 +92,15 @@ private void deleteNarrativeCache(Container container, User user, List<Map<Strin
             }
             catch (IOException e)
             {
-                return cacheData;
+                return eventIds;
```

Comment:
I know it was missing this already, but adding the exception to the errors would be helpful for debugging.

Reply:
Yep